dev/NavigationView/NavigationView.cpp

```diff
@@ -1698,6 +1698,11 @@ void NavigationView::OnSplitViewPaneClosing(const winrt::DependencyObject& /*sen
                     winrt::VisualStateManager::GoToState(*this, L"ListSizeCompact", true /*useTransitions*/);
                     UpdatePaneToggleSize();
                 }
+
+                if (splitView.DisplayMode() == winrt::SplitViewDisplayMode::CompactOverlay || splitView.DisplayMode() == winrt::SplitViewDisplayMode::Overlay)
+                {
+                    winrt::VisualStateManager::GoToState(*this, L"PaneNotOverlay", true /*useTransitions*/);
+                }
```

Comment:
It's very comfusion here.
From your code, Likely PaneNotOverlay and PaneOverlay only happens when DisplayMode is CompactOverlay or Overlay.
So we should have 3 visual states other than 2:
1. DisplayMode is not CompactOverlay or Overlay
2. DisplayMode is ?Overlay, and Pane is closed
3. DisplayMode is ?Overlay, and Pane is opened

Don't use PaneNotOverlay/Overlay, it's easy to mess up with the term: CompactOverlay or Overlay in splitview setting.

It's better to have the LeftNav in the name, for example: Pane???OnLeftNavigationView

Reply:
Added you to the email thread discussing the naming